test/java/com/arpnetworking/metrics/portal/reports/impl/chrome/PdfScreenshotRendererIT.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2019 Dropbox, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.arpnetworking.metrics.portal.reports.impl.chrome;
+
+import com.arpnetworking.metrics.portal.TestBeanFactory;
+import com.arpnetworking.metrics.portal.reports.impl.testing.MockRenderedReportBuilder;
+import com.arpnetworking.metrics.portal.reports.impl.testing.Utils;
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import com.typesafe.config.Config;
+import models.internal.TimeRange;
+import models.internal.impl.PdfReportFormat;
+import models.internal.impl.WebPageReportSource;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mockito;
+
+import java.net.URI;
+import java.time.Instant;
+import java.util.concurrent.CompletionStage;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Tests class {@link PdfScreenshotRenderer}.
+ *
+ * This test-class is only meant to be run manually: it depends on Chrome, which not all environments are guaranteed to have, so this class
+ * is marked {@code @Ignore}. If you want to run it manually, set {@link Utils#CHROME_PATH} as appropriate for your system first.
+ *
+ * @author Spencer Pearson (spencerpearson at dropbox dot com)
+ */
+@Ignore
```

Comment:
Can we make this run on Travis at least?

e.g. If there is an env var that contains the path to Chrome then run the test, otherwise skip it.

https://docs.travis-ci.com/user/chrome
https://docs.travis-ci.com/user/gui-and-headless-browsers/
https://stackoverflow.com/questions/1689242/conditionally-ignoring-tests-in-junit-4